Makefile

```diff
@@ -62,6 +62,17 @@ checklogs:
 .PHONY: pylint
 pylint: requirements .pylint
 
+.PHONY: configgen
+configgen:
```

Comment:
It's alive!

Lol, yeah, I was actually hoping someone else will do it before I need to do it :P